tools/icu/icutrim.py

```diff
@@ -163,17 +163,15 @@ def runcmd(tool, cmd, doContinue=False):
 config=json.load(fi)
 fi.close()
 
-if (options.locales):
-  if not config.has_key("variables"):
-    config["variables"] = {}
-  if not config["variables"].has_key("locales"):
-    config["variables"]["locales"] = {}
+if options.locales:
+  config["variables"] = config.get("variables", {})
+  config["variables"]["locales"] = config["variables"].get("locales", {})
   config["variables"]["locales"]["only"] = options.locales.split(',')
```

Comment:
This file uses 4 space indent so can you use that here too?

(Confusing really since we use 2 space indent everywhere else. Oh well.)

Reply:
Done.